slick/src/main/scala/slick/util/ManagedArrayBlockingQueue.scala

```diff
@@ -96,7 +96,6 @@ class ManagedArrayBlockingQueue[E >: Null <: PrioritizedRunnable](maximumInUse:
         nanos = itemQueueNotFull.awaitNanos(nanos)
       }
       insert(e)
-      return true
```

Comment:
Minor improvement. 

`insert` will always return true at this point, but still, it's better to have the logic in only one place and not assume it will always be `true`.